ulmo/notebooks/Convex_Hull_Overlap.ipynb

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
move this and the rest of the files in `notebooks` into `docs/nb`
and then remove this folder.  Thx!